src/components/Capture/withPrivacyStatement.js

```diff
@@ -0,0 +1,12 @@
+import { h } from 'preact'
+import PrivacyStatement from '../PrivacyStatement'
+import { identity } from '../utils/func'
+
+const withPrivacyStatement = !process.env.PRIVACY_FEATURE_ENABLED ? identity :
```

Comment:
Identity is: `x => x`, a function that returns whatever is passed to it.

In this case it would be a component class, but since it doesn't need to augment it in any way, it will return the same class untouched.